### PR TITLE
MCS-1803 Various Integration Test Updates

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**
1. Added documentation on the integration test "outputs" file schema to the README.
2. Config files for all integration tests now have depth maps and object masks disabled by default, to improve runtime performance; the one test that checks the depth and segmentation data now has its own config file.
3. A new controller (Unity window) is now created for each metadata tier, to stop some timeout issues probably due to memory usage.
4. Added "prefab tests" -- see the README for details. Please note that you'll currently see "no texture color list" failures for agents, TVs, some sofas/chairs, and most antiques, which will be fixed on this PR: https://github.com/NextCenturyCorporation/ai2thor/pull/334

### CODE REVIEW

File: machine_common_sense/scripts/run_scene_with_command_file.py [hunk 1]
```diff
@@ -41,7 +41,8 @@ def main():
 
     controller = mcs.create_controller(
         unity_app_file_path=args.mcs_unity_build_file,
-        config_file_or_dict='./run_scripts_config_with_history.ini'
+        # config_file_or_dict='./run_scripts_config_with_history.ini'
+        config_file_or_dict='./phi_config.ini'
```
Comment (rartiss55) [inline machine_common_sense/scripts/run_scene_with_command_file.py:45]: was this change intentional?

**Review by rartiss55 — APPROVED** [review]
Besides the tests that need the agency-fix branch, integration tests pass.

I tried the prefab tests with the development build (which should have the necessary updates). I did get two failures, but perhaps something that was cached previously didn't get updated properly. Leaving it here in case its not just me:
Final results: 412 successful, 2 failed
Failed scenes:
    antique_chair_1: No texture color list
    antique_sofa_1: No texture color list

File: machine_common_sense/scripts/run_scene_with_command_file.py (hunk 1 above)
Comment (ThomasSchellenbergNextCentury) [reply machine_common_sense/scripts/run_scene_with_command_file.py:45]: I didn't change any of those files. I think something weird happened with a merge? I just merged in the latest from development. Looks like it's fixed now. Thanks for alerting me!

**Comment by ThomasSchellenbergNextCentury** [comment]
@rartiss55 Whoops, doh, I forgot that I had fixed the materials on those two prefabs (the were referring to copies of materials I recently deleted). Thanks for finding that. Here's the PR: https://github.com/NextCenturyCorporation/mcs-private/pull/473

**Comment by rartiss55** [comment]
> @rartiss55 Whoops, doh, I forgot that I had fixed the materials on those two prefabs (the were referring to copies of materials I recently deleted). Thanks for finding that. Here's the PR: [NextCenturyCorporation/mcs-private#473](https://github.com/NextCenturyCorporation/mcs-private/pull/473)

Thanks for fixing it! I tried hardcoding the path for my local build and rerunning the prefab tests, got a couple different failures but I'm chalking that up to something I'm missing locally (the updated antique_* objects passed).

**Review by rartiss55 — APPROVED** [review]

**Review by phile-caci — APPROVED** [review]
Looks good. Ran integration and prefab tests no issues (other than what was already noted about the skintones and the other prefabs)
